src/Action/Tool/Ajax/AcsServiceClientAction.php

```diff
@@ -63,10 +63,10 @@ public function __invoke(Request $request): Response
             $request->get('acsSub'),
             $request->get('acsAction'),
             $date,
-            (int)$request->get('acsAttemptNumber'),
+            (int)$request->request->has('acsAttemptNumber') ?: 1,
```

Comment:
will it not be always `1`?

Reply:
Right. Forgot to change that line before committing.
Thanks, fixed.